### DISCUSSION TITLE
Component to Add and Remove items

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1325] [Proposals]
### Goal

Precisamos criar um botão para adicionar e remover entradas de valores.

### Context

Não sei se existe algum estudo dentro do Mística para um componente com essa funcionalidade, a principio, esse botão vai ser aplicado em Loja com o objetivo de adicionar e remover a quantidade de produtos de uma vitrine dentro do carrinho de compras.

### Impact

Criar esse componente ajudaria na experiência da pessoa pessoa usuária.

### Discussion Questions

Existe algum estudo no Mística para um componente com essa funcionalidade?
Pode ser criado atomicamente?

### Do you have a visual proposal?

https://www.figma.com/file/vFfySe28QCgHv1dNooOw6M/Add-Remove-Button?type=design&node-id=0%3A1&mode=design&t=p9I8YRJTLuMAWLoA-1

**Comment by HectorSancho** [comment]
Hola!

Analizando la propuesta visual, el componente está resuelto mediante un botón y esto impediría mostrar todos los estados necesarios.

Creemos que encaja más en la categoría de controles y podría ser parte de Mística core.

Hemos trabajado una propuesta visual y podemos verlo con detalle el jueves en la weekly.

<img width="1932" alt="Frame 1171277947" src="https://github.com/Telefonica/mistica-design/assets/56122343/0bcc7b27-9a01-4c97-9a2f-7b635ec10009">
<img width="950" alt="Frame 1171277945" src="https://github.com/Telefonica/mistica-design/assets/56122343/ea67e9d3-2407-4035-9e8d-5d1900fdc8ab">

https://www.figma.com/file/anogOC0p7IyR7cN9yLJQFq/Unit-control?type=design&node-id=0%3A1&mode=design&t=DgzHMkg8xfvQxygr-1

**Reply by yceballost** [reply]
@annefranca os encaja la solución que plantea Héctor? Si me confirmas cerramos la discussion y creamos tarea para añadirlo al roadmap

**Reply by annefranca** [reply]
Olá @yceballost 

Se encaixa sim! Lembrando daquele ponto que comentamos quinta-feira passada sobre a área de toque de 40px por 40px, está bem?

**Comment by yceballost** [comment]
Ticket creado para meterlo en el roadmap y branch en figma creada para empezar a plantear las specs. @annefranca os encargáis vosotros de definir esto?

https://www.figma.com/file/3R2UbuRNvPFSS30r8s2mXb/%23XX---Counter-component?type=design&node-id=0-1&mode=design

https://github.com/Telefonica/mistica-design/issues/1364
